Inventory management file uploaded successfully.

samridhi-611 opened the pull request.

samridhi-611 committed:
Added succesfully

intimetec-sakshi-agarwal commented on inventory.c:
```diff
@@ -0,0 +1,248 @@
+    Product *newProduct = &(*inventory)[*count];
+    printf("Enter details of the new product:\n");
+    printf("Product ID: ");
+    scanf("%d", &newProduct->id);
```
No check if input is valid integer
No check for constraints (1 ≤ id ≤ 10000

intimetec-sakshi-agarwal commented on inventory.c:
```diff
@@ -0,0 +1,248 @@
+    getchar();
+
+    printf("Product Name: ");
+    fgets(newProduct->name, NAME_LEN, stdin);
```
No check for empty input

samridhi-611 committed:
Fixed the issues

samridhi-611 committed:
Improve input validation and output formatting

Enhanced input validation for product details including ID, price, and quantity. Improved output formatting for product listings.
